commands.js

```diff
@@ -1,45 +1,44 @@
 'use strict';
 const fs = require('fs');
 const chalk = require('chalk');
-const os = require('os');
-const pathFile = pathFinder();
+const path = require('path');
 
-function pathFinder() {
-  let path = process.env['SHORTC_PATH'] + '\\shortc.json';
-  if (path !== undefined) {
-    return path;
-  } else {
-    path = os.homedir() + '\\shortc.json';
-    return path;
-  }
+const pathFile = getSavePath();
+
+function getSavePath() {
+  const currentDir = path.resolve(__dirname);
```

Comment:
There is a problem with current dir. Suppose I'am located at `/home/user`, and typed:
```bash
$ shortc add ...
```
It will create `/home/user/.shortc.json`. And then, I will do this:
```bash
$ cd docs
$ shortc add ...
```
It will create a new file at `/home/user/docs/.shortc.json`.


And that's not what user expects. The user expects to have static files somewhere in the system. So, to work around the problem we can use `os.homedir()` like a base path. So, for example in my case the file would be always located at `/home/gdobn/.shortc.json`, no matter where I call shortc.